pegasus/sites.v3/code.org/views/ai_curricula_blocks.haml

```diff
@@ -1,20 +1,20 @@
 
-- ai_curricula_overline = [hoc_s(:module_grade_7_12), hoc_s(:module_grade_9_12), hoc_s(:module_grade_6_12), hoc_s(:module_grade_6_12), hoc_s(:module_grade_7_12), hoc_s(:module_grade_3_12)]
+- ai_curricula_overline = [hoc_s(:module_grade_7_12), hoc_s(:module_grade_9_12), hoc_s(:module_grade_6_12), hoc_s(:module_grade_6_12), hoc_s(:module_grade_7_12), hoc_s(:module_grade_3_12), hoc_s(:module_grade_3_12)]
 
-- ai_curricula_title = [hoc_s(:ai_curriculum_title_06), hoc_s(:curriculum_name_csa_ai), hoc_s(:ai_curriculum_title_04), hoc_s(:ai_curriculum_title_01), hoc_s(:ai_curriculum_title_02), hoc_s(:ai_curriculum_title_03)]
+- ai_curricula_title = [hoc_s(:ai_curriculum_title_06), hoc_s(:curriculum_name_csa_ai), hoc_s(:ai_curriculum_title_04), hoc_s(:ai_curriculum_title_01), hoc_s(:ai_curriculum_title_02), hoc_s(:ai_curriculum_title_03), hoc_s(:curriculum_name_dance_party_ai_edition)]
 
-- ai_curricula_img = ["/images/ai/ai-curriculum-societal-impacts.png", "/images/ai/ai-curriculum-computer-vision.png", "/images/ai/ai-curriculum-how-ai-works.png", "/images/ai/ai-curriclum-machine-learning.png", "/images/ai/ai-curriclum-ethics.png", "/images/ai/ai-curriclum-oceans.png"]
+- ai_curricula_img = ["/images/ai/ai-curriculum-societal-impacts.png", "/images/ai/ai-curriculum-computer-vision.png", "/images/ai/ai-curriculum-how-ai-works.png", "/images/ai/ai-curriclum-machine-learning.png", "/images/ai/ai-curriclum-ethics.png", "/images/ai/ai-curriclum-oceans.png", "/images/dance-hoc/dance-party-activity-ai-edition-cropped.png"]
 
-- ai_curricula_desc = [hoc_s(:ai_curriculum_desc_06), hoc_s(:curriculum_desc_csa_ai_short), hoc_s(:ai_curriculum_desc_04), hoc_s(:ai_curriculum_desc_01), hoc_s(:ai_curriculum_desc_02), hoc_s(:ai_curriculum_desc_03)]
+- ai_curricula_desc = [hoc_s(:ai_curriculum_desc_06), hoc_s(:curriculum_desc_csa_ai_short), hoc_s(:ai_curriculum_desc_04), hoc_s(:ai_curriculum_desc_01), hoc_s(:ai_curriculum_desc_02), hoc_s(:ai_curriculum_desc_03), hoc_s(:curriculum_desc_dance_party_ai_edition)]
 
-- ai_curricula_duration = [hoc_s(:module_duration_1_hr), hoc_s(:module_duration_5_hrs_plus), hoc_s(:module_duration_8_hrs), hoc_s(:module_duration_20_hrs), hoc_s(:module_duration_1_hr), hoc_s(:module_duration_1_hr)]
+- ai_curricula_duration = [hoc_s(:module_duration_1_hr), hoc_s(:module_duration_5_hrs_plus), hoc_s(:module_duration_8_hrs), hoc_s(:module_duration_20_hrs), hoc_s(:module_duration_1_hr), hoc_s(:module_duration_1_hr), hoc_s(:module_duration_1_hr)]
 
-- ai_curricula_url = [CDO.studio_url("/s/ai-ethics-2023/lessons/2"), "/curriculum/computer-vision", "/ai/how-ai-works", CDO.studio_url("/s/aiml"), CDO.studio_url("/s/ai-ethics-2023/lessons/1"), CDO.studio_url("/s/oceans")]
+- ai_curricula_url = [CDO.studio_url("/s/ai-ethics-2023/lessons/2"), "/curriculum/computer-vision", "/ai/how-ai-works", CDO.studio_url("/s/aiml"), CDO.studio_url("/s/ai-ethics-2023/lessons/1"), CDO.studio_url("/s/oceans"), CDO.studio_url("/s/dance-ai-2023/reset")]
```

Comment:
out of curiosity, why does dance-ai-2023 use `/reset` and the others don't?

Reply:
I'm not sure, those other links came from a different spec doc. This is what we were told to use for Dance AI ([slack convo](https://codedotorg.slack.com/archives/C05P12U42F5/p1700507071860719)).